recipe/meta.yaml

```
@@ -10,31 +10,26 @@ source:
   sha256: 2558a4d7c3864b6f3211dac823ea79869ae0e9f79fbd1aa6fe03f5848d87054f
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - cmake
+    - cmake <3.20
   host:
     - python
     - pip
-    - numpy
+    - {{ pin_compatible('numpy') }}
```

We shouldn't have `pin_compatible` in `host`
```suggestion
    - numpy
```

---

@xhochy that seems to have broken things for Py3.9. Suggestions?